src/Postmark/PostmarkClient.cs

```diff
@@ -1040,5 +1041,69 @@ public async Task<PostmarkResponse> DeleteWebhookConfigurationAsync(long configu
+        /// <returns>Listing of active Suppressions matching the provided query parameters.</returns>
+        public async Task<PostmarkSuppressionListing> ListSuppressions(PostmarkSuppressionQuery query)
+        {
+            var parameters = new Dictionary<string, object>
+            {
+                ["SuppressionReason"] = query.SuppressionReason,
+                ["Origin"] = query.Origin,
+                ["todate"] = query.ToDate?.ToString("O"),
+                ["fromdate"] = query.FromDate?.ToString("O"),
+                ["emailAddress"] = query.EmailAddress,
+            };
+
+            var messageStreamID = query.MessageStream ?? DEFAULT_TRANSACTIONAL_STREAM;
+            var apiUrl = $"/message-streams/{messageStreamID}/suppressions/dump";
+
+            return await ProcessNoBodyRequestAsync<PostmarkSuppressionListing>(apiUrl, parameters);
+        }
+
+        /// <summary>
+        /// Create Suppressions for the specified recipients.
+        /// </summary>
+        /// <param name="suppressionChanges">List of SuppressionChangeRequest objects that specify what recipients to suppress.</param>
+        /// <param name="messageStream">Message stream where the recipients should be suppressed.
+        /// If not provided, they will be suppressed on the default transactional stream.</param>
+        /// <remarks>Suppressions will be generated with a Customer Origin and will have a ManualSuppression reason.</remarks>
+        /// <returns>The status of the request for each recipient.</returns>
+        public async Task<PostmarkSuppressionBulkRequestResult> CreateSuppressions(IEnumerable<PostmarkSuppressionChangeRequest> suppressionChanges,
+            string messageStream = null)
+        {
+            var body = new Dictionary<string, object> { ["Suppressions"] = suppressionChanges.ToList() };
+
+            var messageStreamID = messageStream ?? DEFAULT_TRANSACTIONAL_STREAM;
+            var apiUrl = $"/message-streams/{messageStreamID}/suppressions";
+
+            return await ProcessRequestAsync<Dictionary<string, object>, PostmarkSuppressionBulkRequestResult>(apiUrl, HttpMethod.Post, body);
+        }
+
+        /// <summary>
+        /// Reactivate Suppressions for the specified recipients.
+        /// </summary>
+        /// <param name="suppressionChanges">List of SuppressionChangeRequest objects that specify what recipients to reactivate.</param>
+        /// <param name="messageStream">Message stream where the recipients should be reactivated.
+        /// If not provided, they will be reactivated on the default transactional stream.</param>
+        /// <remarks>Suppressions will only be reactivated if you have the required authority.</remarks>
+        /// <returns>The status of the request for each recipient.</returns>
+        public async Task<PostmarkSuppressionBulkRequestResult> DeleteSuppressions(
+            IEnumerable<PostmarkSuppressionChangeRequest> suppressionChanges, string messageStream = null)
```

Comment:
It were going to allow the stream to be optional, then let’s make that a constant and put it in the method signature directly. This will make it more obvious what it’s doing in intellisense.